scenarios/12_Leaving_Raedwood.cfg

```diff
@@ -262,6 +280,43 @@ Move Ruvio to signpost to ask him for advice."}
         {INTERROGATION12}
     [/event]
 
+    [event]
+        name=last breath
+        [filter]
+            id=Quivre O Raedbor
+        [/filter]
+        [filter_second]
+            id=Gawen Hagarthen
+        [/filter_second]
+        {MSGW_Ruvio _"Gawen! I thought I told you not to waste your time with that one!"}
+        {MSGW_Gawen _"I know, I was just curious to see if he had anything to say."}
+        {MSG_unit _"Well, I don't! So there!"}
+        [kill]
+            id=Quivre O Raedbor
+            # (no animation here, because it's ok to leave it ambiguous whether he died or just left here)
+        [/kill]
+        {MSGW_Ruvio _"See? What did I tell you?"}
+        {MSGW_Gawen _"(<i>sigh</i>) Fine..."}
+    [/event]
+
+    [event]
+        name=last breath
+        [filter]
+            id=Quivre O Raedbor
+        [/filter]
+        [filter_second]
+            id=Ruvio
+        [/filter_second]
+        {MSGW_Ruvio _"..."}
+        [kill]
+            id=Quivre O Raedbor
+            animate=yes
+        [/kill]
+        {MSGW_Gawen _"You just let him die? Why?"}
```

Comment:
Here too.

Reply:
This seems pretty plainspoken already here, too; I'm not sure how to make it plainer...

Reply:
> You just let him die?
> It seemed like he didn't have anything important to say.
> If you say so.

Reply:
changed in d8d2616